### PR TITLE
NIFI-8937 Show component name and version in configure dialog's title bar

### PR DESCRIPTION
**Pull request by adenes**
Added the component name to the "Configure Processor", "Configure Controller Service" and "Configure Reporting Task" dialogs' title bar.

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with **NIFI-XXXX** where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically `main`)?

- [x] Is your initial contribution a single, squashed commit? _Additional commits in response to PR reviewer feedback should be made on this branch and pushed to allow change tracking. Do not `squash` or use `--force` when pushing to allow for clean monitoring of changes._

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via `mvn -Pcontrib-check clean install` at the root `nifi` folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [x] Have you verified that the full build is successful on JDK 8?
- [ ] Have you verified that the full build is successful on JDK 11?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the `LICENSE` file, including the main `LICENSE` file under `nifi-assembly`?
- [ ] If applicable, have you updated the `NOTICE` file, including the main `NOTICE` file found under `nifi-assembly`?
- [ ] If adding new Properties, have you added `.displayName` in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Review by gresockj — APPROVED** [review]
This works as described:
image
image
image

This could use a review from someone more familiar with the UI code, but at a glance it looks good to me, and will certainly add value to NiFi.

**Comment by mtien-apache** [comment]
Reviewing now...

**Review by mtien-apache — COMMENTED** [review]
Looks good! I verified the component name and version show in the configurations header dialog. 

I only have one minor suggestion to remove some un-used code. Once that's done, I can approve this. Nice work - Thanks!

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/modal/jquery.modal.css [hunk 1]
```diff
@@ -72,17 +72,28 @@
...
-    line-height: 56px;
-    padding-left: 20px;
     text-overflow: ellipsis;
+    line-height: 56px;
+    padding: 0 20px;
+    white-space: nowrap;
+}
+
+.dialog-header-text {
+}
```
[the remark this answers is not in the document]
Comment (adenes) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/modal/jquery.modal.css]: Thanks @mtien-apache for the review, I have removed this block.

**Comment by mtien-apache** [comment]
@adenes The changes look good! Unfortunately, the CI build didn't pass. Once that passes I can approve.

**Review by pvillard31 — APPROVED** [review]

**Comment by pvillard31** [comment]
Merging based on @mtien-apache and @gresockj reviews and since the failed build is unrelated to the change itself. Thanks for this improvement @adenes, this is super useful.

**Pull request closed without merging**